src/lib/rest/httpRequestSend.cpp

```diff
@@ -436,8 +436,8 @@ int httpRequestSendWithCurl
     //
     // NOTE: This log line is used by the functional tests in cases/880_timeout_for_forward_and_notifications/
     //       So, this line should not be removed/altered, at least not without also modifying the functests.
-    //
-    LM_W(("Notification failure for %s:%s (curl_easy_perform failed: %s)", ip.c_str(), portAsString, curl_easy_strerror(res)));
+    //    
+    alarmMgr.notificationError(url, "(curl_easy_perform failed: " + std::string(curl_easy_strerror(res)) + ")");
```

Comment:
Does this work?

Normally, the 'std::string' would be on the **first** string of the addition.
The second string should not need it as std::string has a 'plus-operator' accepting char* ...